setup.py

```diff
@@ -4,75 +4,79 @@
 # All Rights Reserved.
 #
 # This software is subject to the provisions of the BSD-like license at
-# http://www.repoze.org/LICENSE.txt.  A copy of the license should accompany
-# this distribution.  THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL
-# EXPRESS OR IMPLIED WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO,
-# THE IMPLIED WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND
-# FITNESS FOR A PARTICULAR PURPOSE
+# https://pylonsproject.org/license.html. A copy of the license should
+# accompany this distribution.  THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND
+# ALL EXPRESS OR IMPLIED WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED
+# TO, THE IMPLIED WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT,
+# AND FITNESS FOR A PARTICULAR PURPOSE
 #
 ##############################################################################
 
 from setuptools import find_packages, setup
 
-README = open("README.rst").read()
-CHANGES = open("CHANGES.txt").read()
+README = open('README.rst').read()
+CHANGES = open('CHANGES.rst').read()
 
 requires = [
-    "Jinja2>=2.5.0,!=2.11.0,!=2.11.1",
-    "MarkupSafe",
-    "pyramid>=1.3.0",  # pyramid.path.DottedNameResolver
-    "zope.deprecation",
+    'jinja2>=2.5.0,!=2.11.0,!=2.11.1',
+    'markupsafe',
+    'pyramid>=1.3.0',  # pyramid.path.DottedNameResolver
+    'zope.deprecation',
 ]
 
-testing_extras = [
-    "coverage",
-    "nose>=1.2.0",
-    "WebTest",
+tests_require = [
+    'jinja2>=2.5.0,!=2.11.0,!=2.11.1',
+    'pyramid>=1.3.0',  # pyramid.path.DottedNameResolver
+    'webtest',
 ]
+
 docs_extras = [
-    "pylons-sphinx-themes >= 0.3",
-    "Sphinx>=1.7.5",
+    'pylons-sphinx-themes >= 0.3',
+    'sphinx>=1.7.5',
+]
+
+testing_extras = tests_require + [
+    'coverage',
+    'pytest>=5.4.2',
+    'pytest-cov',
 ]
 
 setup(
-    name="pyramid_jinja2",
-    version="2.8",
-    description="Jinja2 template bindings for the Pyramid web framework",
-    long_description=README + "\n\n" + CHANGES,
+    name='pyramid_jinja2',
+    version='2.8',
+    description='Jinja2 template bindings for the Pyramid web framework',
+    long_description=README + '\n\n' + CHANGES,
     classifiers=[
-        "Intended Audience :: Developers",
-        "Programming Language :: Python :: Implementation :: CPython",
-        "Programming Language :: Python :: Implementation :: PyPy",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Framework :: Pyramid",
-        "License :: Repoze Public License",
+        'Intended Audience :: Developers',
```

Comment:
Please continue to use double quotes. Please make sure this projects implements a pyproject.toml which sets up black to format all the source files. 

Unless you set the quoting to Black's default, quoting should NOT be modified.

Reply:
Disclaimer: I use single quotes in all my projects so I will be biased.

pyramid_jinja2 is currently 90% single quotes and 10% double quotes. pyramid is mainly single quotes from what I can see. pyramid black's configuration has "skip-string-normalization" and I imported it as is (like I said in the disclaimer, I wasn't too sad about it). 
To be honest, I would feel weird having a different quote system than pyramid. (And more work.)
But this can be the moment we turn it around and I convert everything to double quotes.

Reply:
Pylons Project projects use (or should use) [Black](https://black.readthedocs.io/en/stable/index.html) to automatically format code. [Black's default for strings is double-quotes](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings).

I think Pyramid 1.10 was the first Pylons Project project to adopt Black on [2018-10-15](https://github.com/Pylons/pyramid/pull/3388). [Deform added Black on 2018-11-12](https://github.com/Pylons/deform/pull/381). This project just hasn't received as much love over the years. We add Black, isort, and other code formatting automation at the speed of volunteers 🐌 .

Reply:
Also to make it easier to merge this PR, we should not run Black on the entire code base in it. We should do that in a separate PR. The few changes made here are fine.